timeseries/setup.py

```diff
@@ -56,10 +56,11 @@
         "black~=23.0",
     ],
     "chronos-openvino": [  # for faster CPU inference in pretrained models with OpenVINO
-        "optimum[openvino,nncf]>=1.17,<1.18",
+        "optimum-intel[openvino,nncf]>=1.16,<1.17",
+        "optimum[openvino,nncf]>=1.18,<1.19",
```

Comment:
Does this place any restrictions on the `pandas` version range?

Reply:
No, doesn't change any of the pandas dependency constraints for now. will start allowing pandas 2.2 once nncf releases 2.9.